### DISCUSSION TITLE
front end is not working. I cloned and tried.

### DISCUSSION THREAD

**Discussion by vijaytjperl** [litmuschaos/litmus#2963] [General]
front end is not working. I cloned and tried. attached is screenshot. please help to advise further. I cloned in ubuntu and tried to access the portal
![1](https://user-images.githubusercontent.com/85989996/124379849-febf6400-dcd6-11eb-92f5-f2985caa6e63.JPG)
![2](https://user-images.githubusercontent.com/85989996/124379850-00892780-dcd7-11eb-8813-f41b63621692.JPG)

**Comment by neelanjan00** [comment]
Hello @vijaytjperl, I can see in the screenshot that you have installed the npm package dependencies using the `npm i` command, did you try the `npm start` command?

**Comment by S-ayanide** [comment]
Hey @vijaytjperl, thanks for trying out Litmus 😃 

To access the local development server (with full access to graphql and authentication) you need to make sure the litmus-portal frontend service and litmus-portal server service is port-forwarded after installation. Once the two services are forwarded to port 3000 and 8080, a simple npm start should start the frontend at port 3001.

If not the dev server (localhost), you can also access the frontend from the litmusportal-frontend service endpoint using {NodeIP}:{PORT}

You can access this development Wiki for a more detailed step by step explanation: https://github.com/litmuschaos/litmus/wiki/Litmus-Portal-Development-Guide
